src/Sepes.Infrastructure/Constants/Auth/AzureRoleDefinitionId.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Sepes.Infrastructure.Constants.Auth
+{
+    public static class AzureRoleDefinitionId
+    {
+        public const string READ = "acdd72a7-3385-48ef-bd42-f606fba81ae7";
```

Comment:
What is this?

Reply:
Ah, it's for giving the user that created the data source access to the storage account in Az. That is the id for the generic role "Read" in Azure